interface/wx/config.h

```diff
@@ -549,6 +549,51 @@ class wxConfigBase : public wxObject
     */
     bool Read(const wxString& key, long* l,
               long defaultVal) const;
+    /**
+        Reads a 64-bit long long value, returning @true if the value was found.
```

Comment:
Perhaps everywhere where there is something like "value was/was not found", "value" should be replaced with "key"? 

I think "key" is already used like this in wxConfigBase docs and using "value" can bit a bit confusing when we also have an argument named "value" as well.

Reply:
I see, I just looked at the first couple of getters in the docs, they used "key".  But they were for `wxString`s, the ones for numbers do you use "value".